scripts/js/lib/api/generateToc.ts

```diff
@@ -31,7 +31,9 @@ type Toc = {
 };
 
 export function generateToc(pkg: Pkg, results: HtmlToMdResultWithUrl[]): Toc {
-  const [modules, items] = getModulesAndItems(results);
+  const [modules, items] = getModulesAndItems(results, {
+    isCApi: pkg.language === "C",
```

Comment:
We now repeat this line twice. A better factoring is probably to have `Pkg.isCApi()` for DRY and better encapsulation

Not blocking